### DISCUSSION TITLE
Flutter 2.2.2 was not logged in REST-API endpoint, but how you got it?

### DISCUSSION THREAD

**Discussion by yahu1031** [leoafarias/sidekick#102] [Q&A] [answered]
As of the Flutter releases REST-API endpoint the latest stable version is `2.2.1`

```json
"current_release": {
    "beta": "b22742018b3edf16c6cadd7b76d9db5e7f9064b5",
    "dev": "fa5883b78e566877613ad1ccb48dd92075cb5c23",
    "stable": "02c026b03cd31dd3f867e5faeb7e104cce174c5f"
},
"releases": [
  ......
  {
      "hash": "02c026b03cd31dd3f867e5faeb7e104cce174c5f",
      "channel": "stable",
      "version": "2.2.1",
      "release_date": "2021-05-27T23:09:04.436458Z",
      "archive": "stable/windows/flutter_windows_2.2.1-stable.zip",
      "sha256": "e2c6184fd01338031214c611ad27235d314ecb03a065426a85098a527925a671"
  }
  ......
  ]
```

But how did you able to manage, getting version `2.2.2`? and I have seen you declaring some `env_var` for the endpoint URI. If that can be shareable, please let us know.

**Comment by leoafarias** [comment, marked as answer]
For some reason flutter team changed the URL of the releases to https://storage.googleapis.com/flutter_infra_release/releases/releases_macos.json from https://storage.googleapis.com/flutter_infra/releases/releases_macos.json

**Reply by yahu1031** [reply]
Perfect thank you so much.
